arm/Microsoft.Cache/redis/.test/deploy.full.test.bicep

```diff
@@ -0,0 +1,31 @@
+var location = 'westeurope'
```

Comment:
Note: The CI environment does not yet support Bicep test files.

Instead the tests expect a `.parameters` folder with JSON-Parameter files with a naming convention `*.parameters.json`.

Reply:
Added